src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -545,17 +627,20 @@ $spacing: 4px;
 
 	&--button-variant-h-grouped {
 		&:first-of-type {
-			border-top-left-radius: var(--border-radius-large);
-			border-bottom-left-radius: var(--border-radius-large);
+			border-top-left-radius: var(--border-radius-nc-radio-switch);
+			border-bottom-left-radius: var(--border-radius-nc-radio-switch);
 		}
 		&:last-of-type {
-			border-top-right-radius: var(--border-radius-large);
-			border-bottom-right-radius: var(--border-radius-large);
+			border-top-right-radius: var(--border-radius-nc-radio-switch);
+			border-bottom-right-radius: var(--border-radius-nc-radio-switch);
 		}
 
-		// avoid double borders between elements
-		& + &:not(&.checkbox-radio-switch--checked) {
-			border-left: 0;
+		// remove borders between elements
+		&:not(:last-of-type) {
+			border-right: 0!important;
+		}
+		&:not(:first-of-type) {
+			border-left: 0!important;
 		}
 		& + &.checkbox-radio-switch--checked {
```

Comment:
I think this rule needs to go. It makes the center tab move when activating the right one.

Reply:
> The rule below was the problem, as it is now longer needed.

Yeah, I just couldn't comment there as it was not edited in this PR 🙈